### PR TITLE
Add team label and cleanup Chart.yaml

### PR DESCRIPTION
**Pull request by piontec**
We own this app and it should be our "go to" example of configuring an app. This adds some missing annotations and `Chart.yaml` attributes.

### CODE REVIEW

File: helm/hello-world-app/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,18 @@
 apiVersion: v2
 name: hello-world-app
-version: "0.1.0"
+version: "0.2.1"
+appVersion: "latest"
```
Comment (rossf7) [inline helm/hello-world-app/Chart.yaml:4]: What does `latest` mean? appVersion is optional so I would just leave this out.

**Review by rossf7 — APPROVED** [review]
Looks good just a query on `appVersion`

File: helm/hello-world-app/Chart.yaml (hunk 1 above)
Comment (piontec) [reply helm/hello-world-app/Chart.yaml:4]: True, on the other hand we recommend setting it... I think I'll just take a longer route and do a proper release of the app first

Comment (MarcelMue) [reply helm/hello-world-app/Chart.yaml:4]: Yeah releasing this here should be better. Having latest seems like a worse example to me than having nothing.

**Pull request merged**